src/main/java/uk/gov/hmcts/reform/draftstore/data/DraftStoreDAO.java

```diff
@@ -156,7 +167,9 @@ public Draft mapRow(ResultSet rs, int rowNumber) throws SQLException {
                 rs.getString("user_id"),
                 rs.getString("service"),
                 rs.getString("document"),
-                rs.getString("document_type")
+                rs.getString("document_type"),
+                rs.getTimestamp("created").toInstant().atZone(ZoneId.systemDefault()),
```

Comment:
We may consider extracting a conversion method. Not that important for now, as there are just two lines like this.